runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -223,12 +223,26 @@ export class RepeatingFieldPageController extends PageController {
       answers?.splice(removeAtIndex, 1);
       answers = { [key]: answers };
       await cacheService.mergeState(request, { [sectionName]: answers });
+
+      //TODO: Quick fix but lets see if we can make the returing happen in one place
+      if (
+        state[sectionName][key] === undefined ||
+        state[sectionName][key]?.length < 1 ||
+        this.isSamePageDisplayMode
+      ) {
+        return h.redirect(`?view=0${form_session_identifier}`);
```

Comment:
Is `view=0` the start page, or the same page that we're currently on?

Does seem a bit hacky, but if it works it works, lgtm 👍

Reply:
Its the page with the components has always been 0 view=0. Its weird but it works, todo for a rework